stix_shifter/stix_transmission/src/modules/aws_cloud_watch_logs/aws_cloud_watch_logs_boto3_client.py

```diff
@@ -0,0 +1,47 @@
+import boto3
+import string
+import random
+
+
+class BOTO3Client:
+    def __init__(self, connection, configuration):
+        host = connection.get('host')
+        region_name = host.split('.')[1]
+        auth = configuration.get('auth')
+        aws_access_key_id = auth.get('aws_access_key_id')
+        aws_secret_access_key = auth.get('aws_secret_access_key')
+        self.log_group_names = configuration.get('log_group_names', {})
```

Comment:
i think we can put the `log_group_names` inside connection object. To my understanding, multiple user can have access to individual log groups. 

So just like region name we can set the `log_group_names` inside options of connection object to make it clean. configuration object will only contain authentication details. So, we can access like this:
`self.log_group_names = connection.get('options',{}).get("log_group_names", {}) 
`

Reply:
It will be updated.

Reply:
done